### PR TITLE
Translate passing-props-to-a-component.md

### LINKED ISSUE #90: Passing Props to a Component

### PR DESCRIPTION
**Pull request by rammba**
Closes #90

### CODE REVIEW

File: src/content/learn/importing-and-exporting-components.md [hunk 1]
```diff
@@ -245,7 +245,7 @@ Na ovoj stranici ste naučili:
 
 <Challenges>
 
-#### Delite komponente dalje {/*split-the-components-further*/}
+#### Deliti komponente dalje {/*split-the-components-further*/}
```
Comment (rammba) [inline src/content/learn/importing-and-exporting-components.md:248]: Using infinitive in the Challenge titles.

File: src/content/learn/describing-the-ui.md [hunk 2]
```diff
@@ -188,7 +188,7 @@ Pročitajte **[Pisanje markup-a sa JSX](/learn/writing-markup-with-jsx)** da bis
 
 ## JavaScript u JSX-u sa vitičastim zagradama {/*javascript-in-jsx-with-curly-braces*/}
 
-JSX vam dozvoljava da pišete markup sličan HTML-u unutar JavaScript fajla, čuvajući logiku prikazivanja i sadržaja na istom mestu. Ponekad ćete želeti da dodate malo JavaScript logike ili da referencirate dinamičko svojstvo unutar tog markup-a. U ovoj situaciji možete koristiti vitičaste zagrade u vašem JSX-u da "otvorite prozor" ka JavaScript-u:
+JSX vam dozvoljava da pišete markup sličan HTML-u unutar JavaScript fajla, čuvajući logiku prikazivanja i sadržaja na istom mestu. Ponekad ćete želeti da dodate malo JavaScript logike ili da referencirate dinamičko polje unutar tog markup-a. U ovoj situaciji možete koristiti vitičaste zagrade u vašem JSX-u da "otvorite prozor" ka JavaScript-u:
```
Comment (rammba) [inline src/content/learn/describing-the-ui.md:191]: I think `polje` is widely used term for property/field, although `svojstvo` is the real translation for `property`.

Comment (lxmarinkovic) [reply src/content/learn/describing-the-ui.md:191]: 👍

**Review by lxmarinkovic — COMMENTED** [review]
lgtm 🚀

**Pull request merged**